supervision/draw/utils.py

```diff
@@ -171,3 +172,58 @@ def draw_text(
         lineType=cv2.LINE_AA,
     )
     return scene
+
+
+def copy_paste(
+    source_image: np.ndarray,
+    source_polygon: np.ndarray,
+    target_image: np.ndarray,
+    scale: float = 1.0,
+    x: int = 0,
+    y: int = 0,
+):
+    """
+    Copy and paste a region from a source image into a target image.
+
+    Attributes:
+        source_image (np.ndarray): The image to be copied.
+        source_polygon (np.ndarray): The polygon area to be copied.
+        target_image (np.ndarray): The image to be pasted into.
+        scale: scale factor to apply to the source image
+        x: x offset to paste into the target image at
+        y: y offset to paste into the target image at
+
+
+    Returns:
+        np.ndarray: The target image with the source image pasted into it.
+    """
+    # cv2.imshow('source', source_image)
+    # cv2.imshow('target', target_image)
+    # # print(source_image.shape)
+
+    output = target_image.copy()
```

Comment:
This will be super slow. Do we want/need to copy image?

Reply:
:+1:  can do in place on the target image instead